requirements/dev.in

```diff
@@ -72,3 +72,6 @@ codespell
 
 # For mocking time
 time-machine
+
+# Needed for tools/summarize-topic
+litellm
```

Comment:
This is missing an end-of-file newline.

Reply:
Oh, for some reason I thought this file didn't need a new line.